utils/lzbench/Makefile

```
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2025 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lzbench
+PKG_VERSION:=2.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/inikep/lzbench.git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=8147278a3322ddfa193de589707a2627c99b3183267e7a1cda7b8a2628edc54a
+
+PKG_MAINTAINER:=Nikita Solianik <gxcreator@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later BSD-3-Clause MIT Zlib
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/lzbench
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=Compression
+  TITLE:=In-memory benchmark for compression algorithms
+  URL:=https://github.com/inikep/lzbench
+  DEPENDS:=+libstdcpp +libpthread
+endef
+
+define Package/lzbench/description
+  lzbench is an in-memory benchmark for open-source compression algorithms.
+  It integrates multiple compression libraries into a single executable and
+  allows for easy comparison of compression speed, decompression speed, and
+  compression ratio.
+endef
+
+# Remove optimization flag from TARGET_CFLAGS
+TARGET_CFLAGS := $(filter-out -O%,$(TARGET_CFLAGS))
+
+MAKE_FLAGS += \
+	CXX="$(TARGET_CXX)" \
+	CC="$(TARGET_CC)" \
+	USER_CFLAGS="$(TARGET_CFLAGS)" \
+	USER_CPPFLAGS="$(TARGET_CPPFLAGS)" \
+	LDFLAGS="$(TARGET_LDFLAGS)" \
+	BUILD_STATIC=0 \
+	DONT_BUILD_BRIEFLZ=1 \
+	DONT_BUILD_BSC=1 \
+	DONT_BUILD_BZIP3=1 \
+	DONT_BUILD_CRUSH=1 \
+	DONT_BUILD_CSC=1 \
+	DONT_BUILD_DENSITY=1 \
+	DONT_BUILD_FASTLZMA2=1 \
+	DONT_BUILD_GIPFELI=1 \
+	DONT_BUILD_GLZA=1 \
+	DONT_BUILD_KANZI=1 \
+	DONT_BUILD_LIBDEFLATE=1 \
+	DONT_BUILD_LIZARD=1 \
+	DONT_BUILD_LZF=1 \
+	DONT_BUILD_LZFSE=1 \
+	DONT_BUILD_LZG=1 \
+	DONT_BUILD_LZHAM=1 \
+	DONT_BUILD_LZLIB=1 \
+	DONT_BUILD_LZMAT=1 \
+	DONT_BUILD_LZO=1 \
+	DONT_BUILD_LZRW=1 \
+	DONT_BUILD_LZSSE=1 \
+	DONT_BUILD_NVCOMP=1 \
+	DONT_BUILD_QUICKLZ=1 \
+	DONT_BUILD_PPMD=1 \
+	DONT_BUILD_SNAPPY=1 \
+	DONT_BUILD_SLZ=1 \
+	DONT_BUILD_TAMP=1 \
+	DONT_BUILD_TORNADO=1 \
+	DONT_BUILD_UCL=1 \
+	DONT_BUILD_WFLZ=1 \
+	DONT_BUILD_YAPPY=1 \
+	DONT_BUILD_ZLING=1 \
+	DONT_BUILD_ZPAQ=1
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		$(MAKE_FLAGS)
+endef
```

Just changing order to make it more readable at least for me.

```suggestion

define Build/Compile
	$(MAKE) -C $(PKG_BUILD_DIR) \
		$(MAKE_FLAGS)
endef

TARGET_CFLAGS := $(filter-out -O%,$(TARGET_CFLAGS))

MAKE_FLAGS += \
	CXX="$(TARGET_CXX)" \
	CC="$(TARGET_CC)" \
	USER_CFLAGS="$(TARGET_CFLAGS)" \
	USER_CPPFLAGS="$(TARGET_CPPFLAGS)" \
	LDFLAGS="$(TARGET_LDFLAGS)" \
	BUILD_STATIC=0 \
	DONT_BUILD_BRIEFLZ=1 \
	DONT_BUILD_BSC=1 \
	DONT_BUILD_BZIP3=1 \
	DONT_BUILD_CRUSH=1 \
	DONT_BUILD_CSC=1 \
	DONT_BUILD_DENSITY=1 \
	DONT_BUILD_FASTLZMA2=1 \
	DONT_BUILD_GIPFELI=1 \
	DONT_BUILD_GLZA=1 \
	DONT_BUILD_KANZI=1 \
	DONT_BUILD_LIBDEFLATE=1 \
	DONT_BUILD_LIZARD=1 \
	DONT_BUILD_LZF=1 \
	DONT_BUILD_LZFSE=1 \
	DONT_BUILD_LZG=1 \
	DONT_BUILD_LZHAM=1 \
	DONT_BUILD_LZLIB=1 \
	DONT_BUILD_LZMAT=1 \
	DONT_BUILD_LZO=1 \
	DONT_BUILD_LZRW=1 \
	DONT_BUILD_LZSSE=1 \
	DONT_BUILD_NVCOMP=1 \
	DONT_BUILD_QUICKLZ=1 \
	DONT_BUILD_PPMD=1 \
	DONT_BUILD_SNAPPY=1 \
	DONT_BUILD_SLZ=1 \
	DONT_BUILD_TAMP=1 \
	DONT_BUILD_TORNADO=1 \
	DONT_BUILD_UCL=1 \
	DONT_BUILD_WFLZ=1 \
	DONT_BUILD_YAPPY=1 \
	DONT_BUILD_ZLING=1 \
	DONT_BUILD_ZPAQ=1
```

---

applied